src/api/style.js

```diff
@@ -295,11 +295,11 @@ export default class Style {
         /**
          * @typedef {object} StyleSpec
          * @property {number} resolution
-         * @property {carto.style.expression.Base} color
-         * @property {carto.style.expression.Base} width
-         * @property {carto.style.expression.Base} strokeColor
-         * @property {carto.style.expression.Base} strokeWidth
-         * @property {carto.style.expression.Base} order
+         * @property {carto.style.expressions.Expression} color
+         * @property {carto.style.expressions.Expression} width
+         * @property {carto.style.expressions.Expression} strokeColor
+         * @property {carto.style.expressions.Expression} strokeWidth
+         * @property {carto.style.expressions.Expression} order
```

Comment:
This is redundant. We do not have carto.source.Source for the Base class, we use carto.source.Base instead. IMO we should do the same here with the expressions.

Reply:
I think this is out of the scope of this PR, just changed an inexistent documented object for the real one ;) 

In any case Im totally against emulating java in javascript including fake abstract "clasess". We can discuss this in the future

Reply:
I think the problem is just the naming of the `Expression` class, in any case, it is outside the PR scope